base/float.jl

```diff
@@ -537,20 +537,6 @@ isnan(x::AbstractFloat) = x != x
 isnan(x::Float16) = reinterpret(UInt16,x)&0x7fff > 0x7c00
 isnan(x::Real) = false
 
-"""
-    isfinite(f) -> Bool
-
-Test whether a number is finite.
-
-# Examples
-```jldoctest
-julia> isfinite(5)
-true
-
-julia> isfinite(NaN32)
-false
-```
-"""
 isfinite(x::AbstractFloat) = x - x == 0
```

Comment:
I should probably remove this method

Reply:
Method is required otherwise I get:
```
...
docs/utils.jl
Base  ───────────error during bootstrap:
LoadError("sysimg.jl", 19, StackOverflowError())
jl_apply_generic at /Users/omus/Development/Julia/latest/src/gf.c:2368
isfinite at ./float.jl:541
isinf at ./float.jl:549
decompose at ./hashing2.jl:186
unknown function (ip: 0x119e3d79f)
jl_apply_generic at /Users/omus/Development/Julia/latest/src/gf.c:0
isfinite at ./float.jl:541
...
isinf at ./float.jl:549
decompose at ./hashing2.jl:186

*** This error might be fixed by running `make clean`. If the error persists, try `make cleanall`. ***
make[1]: *** [/Users/omus/Development/Julia/latest/usr/lib/julia/sys.ji] Error 1
make: *** [julia-sysimg-ji] Error 2
```